lib/node_modules/@stdlib/iter/for-each/README.md

```diff
@@ -54,7 +54,7 @@ function assert( v ) {
 }
 
 var it = iterForEach( array2iterator( [ 1, 2, 3, 4 ] ), assert );
-// returns <Object>
+// returns {}
```

Comment:
In this case, keeping `<Object>` is better. `{}` would indicate that a plain object is returned.
```suggestion
// returns <Object>
```